Library/Homebrew/livecheck/strategy/pypi.rb

```diff
@@ -44,10 +46,8 @@ def self.match?(url)
           URL_MATCH_REGEX.match?(url)
         end
 
-        # Extracts information from a provided URL and uses it to generate
-        # various input values used by the strategy to check for new versions.
-        # Some of these values act as defaults and can be overridden in a
-        # `livecheck` block.
+        # Extracts the package name from the provided URL and generates the
+        # PyPI JSON API endpoint.
```

Comment:
```suggestion
        # PyPI JSON API URL.
```

There may be something to be said about using "URL" language here, for the sake of being explicit and aligning with the `values[:url]` name.